bundle/edu.gemini.ui.workspace/src/main/java/edu/gemini/ui/workspace/util/SimpleInternalFrame.java

```diff
@@ -320,7 +274,7 @@ public void setSelected(boolean newValue) {
      * @param bar     the panel's tool bar
      * @return the panel's built header area
      */
-    private JPanel buildHeader(JLabel label, JToolBar bar, Action helpAction, Icon helpIcon) {
+    private JPanel buildHeader(JLabel label, JToolBar bar, Action helpAction, Icon helpIcon, String hint) {

```

Comment:
That's something I've also been wondering about. In most new code I write, I use Java 8 `Optional`, and in old code, I keep Gemini `Option`, and nobody seems to have complained thus far :smile: